docs/UserGuide.md

```diff
@@ -468,7 +471,7 @@ A quick overview of all supported commands, their formats and examples are given
 | **CURL** | <a name="glossary-curl"></a> CURL is short for **Client URL** and is a command-line tool used in the transfer of data via different network protocols |
 | **Index** | Index in this guide refers to the position of the endpoint in the endpoint list (represented by the number beside the endpoint) |
 | **Protocol** | <a name="glossary-protocol"></a> A protocol is a system of rules that define how data is exchanged within or between systems |
-
+| **int**| <a name="glossary-int"></a> A primitive data type of Java that has the maximum value of (2^31)-1. The minimum value an `int` can have is -(2^31).|
```

Comment:
should we add what a list is also?

Reply:
I think depends on what context do you use `list` in, right now it seems to be ok without it.

Reply:
ok i think can just merge then

Reply:
yea more can be added in the next update